pkg/analyze/common_status.go

```diff
@@ -104,7 +105,7 @@ func commonStatus(outcomes []*troubleshootv1beta2.Outcome, name string, iconKey
 			if exists == false && outcome.Pass.When != "absent" {
 				result.IsFail = true
 				result.Message = fmt.Sprintf("The %s %q was not found", resourceType, name)
-				result.URI = outcome.Fail.URI
```

Comment:
is it worth doing an `if outcome.Pass != nil { result.URI = outcome.Pass.URI }` or something here? just to be super safe...? since we're talking about data coming back potentially as JSON content?

in theory it should always be populated the way we'd expect, but defensive programming and all :) especially since we've got a nil pointer reference we can't repro and all, which could be environment specific somehow

Reply:
@CpuID isn't this condition checked in line [103 above](https://github.com/replicatedhq/troubleshoot/blob/c079d96efedbdac9033547e56aff912910167a2b/pkg/analyze/common_status.go#L103)? Unless I'm missing something here.